src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -106,7 +109,7 @@ var createFiber = function(tag : number, key : null | string) : Fiber {
     memoizedInput: null,
     output: null,
 
-    hasPendingChanges: true,
+    pendingWorkPriority: 0,
```

Comment:
If so, then it is probably the tags that needs to be more restrictive. E.g. not allowing ordinal comparison.